Assets/Scripts/AsteroidCollisionHealthEffect.cs

```diff
@@ -1,16 +1,14 @@
-﻿using UnityEngine;
+﻿using JetBrains.Annotations;
+using UnityEngine;
 
 public class AsteroidCollisionHealthEffect : HealthEffectBehavior
 {
-    public override int ComputeStrength(Collision collision = null) => collision == null
+    public override int ComputeStrength([CanBeNull] Collision collision = null) => collision == null
         ? 0
-        : Mathf.RoundToInt(Mathf.Clamp(collision.impulse.magnitude / Time.fixedDeltaTime * (damageMultiplier / 1000),
-            1f,
-            maxDmg));
+        : Mathf.RoundToInt(Mathf.Clamp(collision.impulse.magnitude / Time.fixedDeltaTime / 1000 * damageMultiplier,1f, maxDmg));
```

Comment:
I meant move maxDmg to the same line as 1f but keep them on a separate line from the mathy one, but oh well

Reply:
🤷‍♂️